src/cirrus/plusone.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+"""
+_plusone_
+
+repo/cirrus independent util for flagging PRs in github as plusone'd
+
+"""
+import sys
+import json
+import argparse
+import requests
+from configuration import get_github_auth
+
+
+class GitHubHelper(object):
+    """
+    GitHubHelper
+
+    Lightweight helper for this command to set up
+    a GH session
+
+    """
+    def __init__(self):
+        self.user, self.token = get_github_auth()
+        self.auth_headers = {
+            'Authorization': 'token {0}'.format(self.token),
+            'Content-Type': 'application/json'
+        }
+        self.session = requests.Session()
+        self.session.headers.update(self.auth_headers)
+
+    def get_pr(self, org, repo, pr_id):
+        """
+        _get_pr_
+
+        grab the PR details
+        """
+        url = "https://api.github.com/repos/{org}/{repo}/pulls/{id}".format(
+            org=org,
+            repo=repo,
+            id=pr_id
+        )
+        resp = self.session.get(url)
+        resp.raise_for_status()
+        data = resp.json()
+        return data
+
+    def plus_one(self, org, repo, sha, context, issue_url):
+        """
+        _plus_one_
+
+        Set the status for the given context to success on the
+        provided sha
+        """
+        url = "https://api.github.com/repos/{org}/{repo}/statuses/{sha}".format(
+            org=org,
+            repo=repo,
+            sha=sha
+        )
+        data = json.dumps(
+            {
+                "state": 'success',
+                "description": "+1 by {}".format(self.user),
```

Comment:
Would `"description": "{} by {}".format(context, self.user)` make more sense, as +1 is just the default context?